c2corg_ui/static/js/addassociationdirective.js

```diff
@@ -55,7 +55,13 @@ app.AddAssociationController = function(appApi, $attrs) {
   this.api_ = appApi;
 
 
-  this.dataset = $attrs['dataset'];
+  /**
+   * @private
+   * @type {string}
+   */
+  this.dataset_ = $attrs['dataset'];
```

Comment:
Isn't it a job for ``@struct`` and simple "dot" notation? As you did for ``parentId`` and ``addedDocuments``.

I think you should simply write
```
/**
 * @type {string}
 * @export
 */
this.dataset;
```